services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,  # movie=movie_id
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    if session_date:
+        formated = datetime.strptime(session_date, "%Y-%m-%d").date()
```

Comment:
The `session_date` parameter is expected to be a `datetime` object, but the code attempts to parse it as a string. Ensure that `session_date` is passed as a `datetime` object or adjust the logic accordingly.